config_samples/base/testnet/hoodi_vaults_testnet_config.json

```diff
@@ -0,0 +1,439 @@
+{
+  "unverified_contracts": {
+    "0x4d0E05677c1005C4C808cb5e824abE99b2b39497": "N/A",
+    "0x49335f718631a7aA201B459B499AFfdc1d8Ca774": "N/A",
+    "0xfF772cd178D04F0B4b1EFB730c5F2B9683B31611": "N/A",
+    "0x5ED5661F744E11A2076865810B0d7553b74468D2": "N/A",
+    "0x5A8bF7fe99a69c2b34B540eA09d4a48c6ae78C21": "N/A"
+  },
+  "contracts": {
```

Comment:
What is the ordering approach used here? Would rather want to have the contracts to be grouped like here:
- https://docs.lido.fi/deployed-contracts/
- https://docs.lido.fi/deployed-contracts/hoodi-lidov3
- https://docs.lido.fi/deployed-contracts/holesky